tests/Unit/EnvServiceTest.php

```
@@ -0,0 +1,114 @@
+
+use Bigpixelrocket\DeployerPHP\Services\EnvService;
+use Symfony\Component\Filesystem\Filesystem;
+
+//
+// Test Helpers
+// -------------------------------------------------------------------------------
+
+function mockFilesystem(bool $exists = true, string $content = '', bool $throwError = false): Filesystem
+{
+    return new class ($exists, $content, $throwError) extends Filesystem {
+        public function __construct(private bool $exists, private string $content, private bool $error)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$error'. (undefined)

(UnusedFormalParameter)